pkg/ingester/ingester.go

```diff
@@ -1107,8 +1112,7 @@ func (i *Ingester) Series(ctx context.Context, req *logproto.SeriesRequest) (*lo
 }
 
 func (i *Ingester) GetStats(ctx context.Context, req *logproto.IndexStatsRequest) (*logproto.IndexStatsResponse, error) {
-	sp, ctx := opentracing.StartSpanFromContext(ctx, "Ingester.GetStats")
```

Comment:
This span is redundant with the `/logproto.Querier/GetStats` span. We won't be losing any information by removing it but we'll end up with a way more concise/smaller trace.
![image](https://github.com/grafana/loki/assets/7079397/f892165a-3d69-4e69-a7f5-bce689bd6d8a)